ci/build_docs.sh

```diff
@@ -5,3 +5,10 @@ set -x #echo on
 cd sphinx
 export GOOGLE_API_KEY=${GOOGLE_API_KEY:-"unset"}
 make SPHINXOPTS=-v all
+
+if grep -e "UA-27761864-7" build/html/index.html > /dev/null; then
```

Comment:
FYI this ID will have to change sometime in the next year for "Universal Analytics" transition. We could search for a code snippet instead, but I'm not sure if there is some consistent code snippet we could search for either

Reply:
Good point - yes, that won't be easy to future proof... I'd try `grep -e 'src="https://www.google-analytics.com' build/html/index.html` for now!